ui/src/app/shared/components/tri-state-button.component.ts

```diff
@@ -1,9 +1,9 @@
-import {Component,Input,Output,OnInit, AfterViewInit, DoCheck,
-  ViewChild,EventEmitter,ChangeDetectionStrategy,
+import { Component, Input, Output, OnInit, AfterViewInit, DoCheck,
```

Comment:
@cppwfs Yes, it might be a good idea to fine-tune things further. E.g. order of imports, method orders etc.